src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -190,6 +191,37 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
     }
     CATCH_RETURN();
 
+    ConptyConnection::ConptyConnection(const uint64_t hSig,
+                                       const uint64_t hIn,
+                                       const uint64_t hOut,
+                                       const uint64_t hClientProcess) :
+        _initialRows{ 25 },
+        _initialCols{ 80 },
+        _commandline{ L"" },
+        _startingDirectory{ L"" },
+        _startingTitle{ L"" },
+        _environment{ nullptr },
+        _guid{},
+        _u8State{},
+        _u16Str{},
+        _buffer{},
+        _inPipe{ (HANDLE)hIn },
```

Comment:
should these be reinterpret_casts?

Reply:
Sure I can make all 3 of these reinterpret_cast and suppress the analyzer warning since we have to do a "dangerous" cast here.

Reply:
Toast with the implementation call.